snapd-qt/assertion.cpp

```diff
@@ -19,26 +19,32 @@ QSnapdAssertion::QSnapdAssertion (const QString& contents, QObject *parent) :
 
 QStringList QSnapdAssertion::headers () const
 {
-    g_auto(GStrv) headers = NULL;
+    GStrv headers = NULL;
     QStringList result;
 
     headers = snapd_assertion_get_headers (SNAPD_ASSERTION (wrapped_object));
     for (int i = 0; headers[i] != NULL; i++)
         result.append (headers[i]);
+
+    g_strfreev (headers);
```

Comment:
This change is exactly what was done before with `g_auto(GStrv)`...

Reply:
Ops. You're right. I guess it's a leftover. Removing!